### PR TITLE
docs: add ops tasks procedure

### PR DESCRIPTION
**Pull request by fameal**
Add ops tasks procedure for:
- Deployment of an experimental vault
- Scale-up until production-ready

### CODE REVIEW

File: docs/OPERATIONS.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+**IMPORTANT**: you should **NOT** create a new release with a test vault
+
+## Before deploying
+- [ ] Coordinate with Core Dev strategist for getting a review in board https://github.com/orgs/iearn-finance/projects/5
+- [ ] Peer review completed by:
+    - yearn.rocks: at least 2 strategists
+    - Experimental: at least 1 strategist + 1 core dev in board 
+- [ ] Check if want token has a deploy vault already (>=v0.3.0) and coordinate to use that first if possible.
+- [ ] (only if a new vault is needed) Deploy using the registry.
```
Comment (lehnberg) [inline docs/OPERATIONS.md:14]: Question: Given the room to make errors in parameters and the need for us to verify that those are correct in any case, should the Strategist really be doing the vault deployment, or is it better this is handled by dedicated people?

File: docs/OPERATIONS.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    - `registry.newExperimentalVault(want, you, multisig, treasury, "", "")`
+        - args: token, gov, guardian, rewards, name, symbol
+        - multisig (strategists) = '0x16388463d60FFE0661Cf7F1f31a7D658aC790ff7'
+        - gov for testing = '0x846e211e8ba920b353fb717631c015cf04061cc9' (dev.ychad.eth) 
+        - treasury = '0x93A62dA5a14C80f265DAbC077fCEE437B1a0Efde'
+        (treasury.ychad.eth)
+    - [ ] Coordinate with core devs proper cap limit for new vault and other settings
```
Comment (lehnberg) [inline docs/OPERATIONS.md:21]: Are caps arbitrary? If so, maybe mention what the criteria to decide on limits are. 

If possible, would be even better to have some kind of table for what types of limits we set, and how to progress to the next stage.

File: docs/OPERATIONS.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+    - `strategy.setRewards(address)`
+    Read below if you want to use the sharer contract
+- [ ] Set management fee to 0
+    - `vault.setManagementFee(0)`
+- [ ] Set governance
+    - `vault.setGovernance(multisig)`
+    - multisig = '0x846e211e8ba920b353fb717631c015cf04061cc9'
+    (dev.ychad.eth)
+    - Governance needs to be accepted before it is in place. After you set this you will still have power until part 2.
+- [ ] Run tests against "live" vault and strategy in mainnet-fork
```
Comment (lehnberg) [inline docs/OPERATIONS.md:48]: Can we describe a set of standard tests here that we want to make sure work as they should for every vault/strategy?

Comment (lehnberg) [reply docs/OPERATIONS.md:48]: I see some tests are described further below, perhaps reference them in this section?

**Review by lehnberg — COMMENTED** [review]
Nice work on this, added some comments and feedback → do with it as you please ✌️

File: docs/OPERATIONS.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+- [ ] Check if want token has a deploy vault already (>=v0.3.0) and coordinate to use that first if possible.
+- [ ] (only if a new vault is needed) Deploy using the registry.
+    - `registry.newExperimentalVault(want, you, multisig, treasury, "", "")`
+        - args: token, gov, guardian, rewards, name, symbol
+        - multisig (strategists) = '0x16388463d60FFE0661Cf7F1f31a7D658aC790ff7'
+        - gov for testing = '0x846e211e8ba920b353fb717631c015cf04061cc9' (dev.ychad.eth) 
+        - treasury = '0x93A62dA5a14C80f265DAbC077fCEE437B1a0Efde'
+        (treasury.ychad.eth)
+    - [ ] Coordinate with core devs proper cap limit for new vault and other settings
+    - [ ] Check new vault has ABI setup on etherscan (If Verification with vyper and proxy is not working that is ideal)
```
[the remark this answers is not in the document]
Comment (fameal) [reply docs/OPERATIONS.md:22]:
```suggestion
    - [ ] Check new vault has ABI setup on Etherscan (until verification using Vyper and proxy works on Etherscan)
```

File: docs/OPERATIONS.md [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+NOTE: This repo is encouraged to create multiple scripts for governance and dev multisig execution of complex transactions
+https://github.com/iearn-finance/chief-multisig-officer
+
+**IMPORTANT**: you should **NOT** create a new release with a test vault
+
+## Before deploying
+- [ ] Coordinate with Core Dev strategist for getting a review in board https://github.com/orgs/iearn-finance/projects/5
+- [ ] Peer review completed by:
+    - yearn.rocks: at least 2 strategists
+    - Experimental: at least 1 strategist + 1 core dev in board
```
[the remark this answers is not in the document]
Comment (fameal) [reply docs/OPERATIONS.md:12]:
```suggestion
    - Experimental: at least 1 strategist + 1 core dev in the Code Review board 
```

File: docs/OPERATIONS.md (hunk 1 above)
Comment (fameal) [reply docs/OPERATIONS.md:14]: It's a great question. I've added this to my list of tasks to revisit when we have all the process documented. I'd like to have the current process, with a nice amount of details and then we can improve it. What do you think?

File: docs/OPERATIONS.md (hunk 2 above)
Comment (fameal) [reply docs/OPERATIONS.md:21]: Caps are more or less arbitrary. Nevertheless, we need this you are suggesting. Will add. Nice catch.

Comment (fameal) [reply docs/OPERATIONS.md]: Caps added.

File: docs/OPERATIONS.md (hunk 3 above)
Comment (fameal) [reply docs/OPERATIONS.md]: Tests described and also op tests added to the brownie-mix :)

**Comment by lehnberg** [comment]
Nice improvements 👍

File: docs/OPERATIONS.md [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+    - `registry.endorseVault(strategy)` Needs ychad.eth governace accepted for this to work
+    - **Order is important. Will fail if order is wrong**
+- [ ] Now you are on main yearn page!
+
+## Limits per Stage
+These are the stantdard deposit limits per stage. This can be increased or decreased on a case by case basis.
+
+| Stage | Limit |
+| ---- | --- |
+| Yearn.rocks / Experimental | $500.000 |
```
Comment (lehnberg) [inline docs/OPERATIONS.md:109]: $500.000 → $500,000 (or $500K)
$10.000.000 → $10,000,000 (or $10M)

File: docs/OPERATIONS.md [hunk 7]
```diff
@@ -0,0 +1,116 @@
...
+    - `vault.setManagementFee(200)`
+- [ ] Set governance to ychad.eth
+- [ ] yearn governance now must accept governance and endorse
+    - `strategy.acceptGovernance()` should be run by ychad.eth
+    - `registry.endorseVault(strategy)` Needs ychad.eth governace accepted for this to work
+    - **Order is important. Will fail if order is wrong**
+- [ ] Now you are on main yearn page!
+
+## Limits per Stage
+These are the stantdard deposit limits per stage. This can be increased or decreased on a case by case basis.
```
Comment (lehnberg) [inline docs/OPERATIONS.md:105]: *standard

Comment (lehnberg) [inline docs/OPERATIONS.md:105]: *They can be adjusted on a case by case basis.

File: docs/OPERATIONS.md [hunk 8]
```diff
@@ -0,0 +1,116 @@
...
+- Tag vault as "experimental" in v2.registry.ychad.eth
+    - `registry.tagVault(vaultAddr, "https://meta.yearn.network/vaults/${vaultAddr}/vault.json")`
+
+### Sharer contract (optional)
+Sharer is a contract for distributing strategist rewards. For boarding school graduates suggested is 34% to strategist_ms and 66% to strategist.
+
+Repo: https://github.com/Grandthrax/Sharer
+
+- Setup rewards for your strategy by calling `sharer.addContributors`
+- If you forked someone elses strategy then cut them in
```
Comment (lehnberg) [inline docs/OPERATIONS.md:64]: *someone else's

**Review by lehnberg — APPROVED** [review]
Approving with minor nit-picks as per comments

File: docs/OPERATIONS.md [hunk 9]
```diff
@@ -0,0 +1,274 @@
...
+- [ ] Deposit some `want` tokens into the vault.
+- [ ] Do first `harvest` and make sure it worked correctly.
+
+  ```python
+  strategy.harvest()
+  ```
+
+- [ ] Publish on yearn.rocks:
+  - Governance dev multisig **_must_** call `vault.acceptGovernance()` first.
+  - Talk to Facu.
```
Comment (milkyklim) [inline docs/OPERATIONS.md:141]: How do I contact him?

File: docs/OPERATIONS.md [hunk 10]
```diff
@@ -0,0 +1,274 @@
...
+
+## Setting up Keep3r
+
+- [ ] Adjust trigger variables until they are correct:
+  - `strategy.setProfitFactor()`
+  - `strategy.setDebtThreshold()`
+  - `strategy.setMaxReportDelay()`
+- [ ] Add strategy to the keep3r job:
+  - `keep3r_manager.addStrategy(strategy, 1_500_000, 1_500_000)`
+  - Tell Poolpi Tako `harvest` and `tend` gas usage. For instance 1.5m.
```
Comment (milkyklim) [inline docs/OPERATIONS.md:152]: How do I find him?

File: docs/OPERATIONS.md [hunk 11]
```diff
@@ -0,0 +1,274 @@
...
+"Sharer" is a contract for distributing/splitting strategist rewards. For boarding school graduates suggested split is 34% to strategist multisig and 66% to strategist – [repo](https://github.com/Grandthrax/Sharer).
+
+- Setup rewards for your strategy by calling `sharer.addContributors`.
+- Include devs if you forked someone else's strategy.
+- Be sure to reward people who helped you.
+
+### Example Script
+
+```python
+# TBD Fill CMO example script to fill
```
Comment (milkyklim) [inline docs/OPERATIONS.md:127]: `TODO:`

File: docs/OPERATIONS.md [hunk 12]
```diff
@@ -0,0 +1,274 @@
...
+- Coordinate with Core Dev strategist for getting a review on [board](https://github.com/orgs/iearn-finance/projects/5).
+- Complete peer review by at least 2 strategists.
+- Check if `want` token has a deployed vault already (>=v0.3.0) and coordinate to use that first if possible.
+- If a new vault is needed, deploy it using the registry:
+
+  ```python
+  registry.newExperimentalVault(want, you, multisig, treasury, "", "")
+  ```
+
+  - args: token, gov, guardian, rewards, name, symbol
```
Comment (milkyklim) [inline docs/OPERATIONS.md:42]: What are those?

File: docs/OPERATIONS.md [hunk 13]
```diff
@@ -0,0 +1,274 @@
...
+  - `multisig` (strategists) = '0x16388463d60FFE0661Cf7F1f31a7D658aC790ff7'
+  - gov for testing = '0x846e211e8ba920b353fb717631c015cf04061cc9' (dev.ychad.eth)
```
Comment (milkyklim) [inline docs/OPERATIONS.md:44]: And where is this one coming from?

**Review by milkyklim — CHANGES_REQUESTED** [review]

File: docs/OPERATIONS.md (hunk 9 above)
Comment (fameal) [reply docs/OPERATIONS.md]: Removed. Added instructions to create a PR.

File: docs/OPERATIONS.md (hunk 10 above)
Comment (fameal) [reply docs/OPERATIONS.md]: Removed. Added instructions to create a PR. Needs new Keep3r jobs deployed, TBD soon.

File: docs/OPERATIONS.md (hunk 12 above)
Comment (fameal) [reply docs/OPERATIONS.md:43]: >  - args: token, gov, guardian, rewards, name, symbol
Is the description of the arguments for this function: registry.newExperimentalVault()

Comment (fameal) [reply docs/OPERATIONS.md:43]: Rewrote it in a more script-like style

File: docs/OPERATIONS.md (hunk 13 above)
Comment (fameal) [reply docs/OPERATIONS.md:45]: That's the data to replace the variable names here:

registry.newExperimentalVault(want, you, multisig, treasury, "", "")

Comment (fameal) [reply docs/OPERATIONS.md:45]: Rewrote it in a more script-like style

File: docs/OPERATIONS.md (hunk 11 above)
Comment (milkyklim) [reply docs/OPERATIONS.md:132]: Will be added in next PR.

**Review by milkyklim — APPROVED** [review]

**Pull request merged**